packages/snaps-simulator/src/features/builder/components/TemplateComponentList.tsx

```diff
@@ -33,57 +44,69 @@ const SVG = `<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="
 
 const TEMPLATE_COMPONENTS: TemplateComponent[] = [
   {
-    icon: 'panel',
-    text: 'Panel',
-    data: panel([]),
+    icon: 'box',
+    text: 'Box',
+    data: Box({ children: null }),
     droppable: true,
   },
   {
     icon: 'heading',
     text: 'Heading',
-    data: heading('Heading'),
+    data: Heading({ children: 'Heading' }),
     droppable: false,
   },
   {
     icon: 'text',
     text: 'Text',
-    data: text('Text'),
+    data: Text({ children: 'Text' }),
     droppable: false,
   },
   {
     icon: 'divider',
     text: 'Divider',
-    data: divider(),
+    data: Divider({}),
     droppable: false,
   },
   {
     icon: 'copyable',
     text: 'Copyable',
-    data: copyable('Copyable text'),
+    data: Copyable({ value: 'Text to copy' }),
     droppable: false,
   },
   {
     icon: 'image',
     text: 'Image',
-    data: image(SVG),
+    data: Image({ src: SVG }),
     droppable: false,
   },
   {
     icon: 'button',
     text: 'Button',
-    data: button('Button'),
+    data: Button({ children: 'Button' }),
     droppable: false,
   },
   {
     icon: 'form',
     text: 'Form',
-    data: form('form', []),
+    data: Form({
+      name: 'form',
+      children: [] as unknown as FormElement['props']['children'],
```

Comment:
Why do you need this?

Reply:
It's just to silence the error due to invalid children. I saw that we allow null now. Will look at changing this

Reply:
We reverted the change that components must have >0 children, so I think you can just remove the type cast now.